application/comit_node/src/libp2p_bam/behaviour.rs

```diff
@@ -141,7 +141,7 @@ where
     }
 
     fn inject_connected(&mut self, peer_id: PeerId, endpoint: ConnectedPoint) {
-        log::debug!(target: "bam", "connected to {} at {:?}", peer_id, endpoint);
+        log::debug!(target: "sub-libp2p", "connected to {} at {:?}", peer_id, endpoint);
```

Comment:
Conceptually I agree.
I would do the change from `bam` -> `comit` once the RFC is written and we fleshed the terminology out.

`sub-libp2p` is supposed to stand for `subsystem-libp2p` since `libp2p` is already taken by the library.
Should we change it to `network`?